examples/mxnet_simple.py

```diff
@@ -0,0 +1,128 @@
+"""
+Optuna example that optimizes multi-layer perceptrons using MX-NET.
```

Comment:
Please replace 'MX-NET' with 'MXNet' as can be seen in [the official site](https://mxnet.apache.org/).

Reply:
Yeah. I missed it I guess.